src/Table.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { forwardRef, memo, type ReactNode, type CSSProperties } from "react";
+import { assert } from "tsafe/assert";
+import type { Equals } from "tsafe";
+import { fr } from "./fr";
+import { cx } from "./tools/cx";
+import { symToStr } from "tsafe/symToStr";
+import type { FrClassName } from "./fr/generatedFromCss/classNames";
+
+export type TableProps = {
+    fixed?: boolean;
+    data: ReactNode[][];
+    className?: string;
+    caption?: ReactNode;
+    bordered?: boolean;
+    headers?: ReactNode[];
+    noScroll?: boolean;
+    noCaption?: boolean;
```

Comment:
There is no need for a `noCaption` prop, we should just test if the caption is undefined.

Reply:
If I'm not mistaking, as mentioned [here](https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/tableau) for some RGAA purposes, the well named `no-caption` option should only visually hide the so called caption and not prevent its rendering.

> Il existe un modifier fr-table--no-caption permettant de cacher (visuellement) le caption afin de l’utiliser uniquement dans le cadre de l’accessibilité.